scripts/byron-to-alonzo/mkfiles.sh

```diff
@@ -312,7 +313,7 @@ cp ../configuration/cardano/shelley_qa-alonzo-genesis.json shelley/genesis.alonz
 # We're going to use really quick epochs (300 seconds), by using short slots 0.2s
 # and K=10, but we'll keep long KES periods so we don't have to bother
 # cycling KES keys
-sed -i shelley/genesis.spec.json \
+sed -i~ \
```

Comment:
This now creates two `genesis.spec.json` files which makes things ambiguous

Reply:
done